### PR TITLE
[zh] Sync English site changes for authentication ref

### PR DESCRIPTION
**Pull request by tengqm**
Some changes are large because ... the original translator failed to include the English source. I don't know why those PRs got merged in the first place.

### CODE REVIEW

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 1]
```diff
@@ -1,89 +1,233 @@
...
+### Token 控制器    {#token-controller}
+
+TokenController 作为 `kube-controller-manager` 的一部分运行，以异步的形式工作。
+其职责包括：
+
+- 监测 ServiceAccount 的创建并创建相应的服务账号令牌 Secret 以允许访问 API。
+- 监测 ServiceAccount 的删除并删除所有相应的服务账号令牌 Secret。
+- 监测服务账号令牌 Secret 的添加，保证相应的 ServiceAccount 存在，如有需要，
+  向 Secret 中添加令牌。
+- 监测服务账号令牌 Secret 的删除，如有需要，从相应的 ServiceAccount 中移除引用。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/reference/access-authn-authz/service-accounts-admin.md:164]: 加上前缀更明确一些。

**Pull request merged**